app/views/resources/show.html.erb

```diff
@@ -21,6 +21,7 @@
                          edit_path: backend.url_for([:edit, @media_content]) }
                       ]
                       end
+  users = resource_users(@media_content)
```

Comment:
I tried to do with the model, but for some reason it did't work (method not found or something like that)... I've tried again and it works perfect 🙃